UnityProject/Assets/Scripts/Equipment/ObjectPool.cs

```diff
@@ -14,13 +14,16 @@ public class ObjectPool : MonoBehaviour
 
 		public void AddGameObject(GameObject obj)
 		{
-			obj.transform.position = transform.position;
-			obj.transform.parent = transform;
+//            obj.transform.position = transform.position;
+//            obj.transform.parent = transform;
+			var objTransform = obj.GetComponent<CustomNetTransform>();
+			objTransform.DisappearFromWorldServer();
+//            objTransform.SetParent(transform);

```

Comment:
is that commented out code needed? otherwise would be better to just remove it.

Reply:
As You see above, I tend to agree...
If we ever need to look back we just git it.